packages/docs/src/components/layout/Layout.tsx

```diff
@@ -80,6 +80,23 @@ const Layout = ({
           lang: 'en',
         }}
       >
+        <meta
+          property="og:title"
+          content={slug?.includes('not-in-sidebar') ? baseTitle : frontmatter?.title}
```

Comment:
Should the title also include the context of the CMS design system like the actual tab title does? When I go to a page like Alert, the title is just `Alert` for this meta tag, but the tab itself gives some better context with `Alert - CMS Design System`.

Reply:
oh that's a good idea!